File: Java/_171_Excel_Sheet_Column_Number/Solution.java
class Solution {
    public int titleToNumber(String s) {
        if (s.length() == 1)
			return s.charAt(0) - 65 + 1;
		return s.charAt(s.length() - 1) - 65 + 1 + titleToNumber(s.substring(0, s.length() - 1)) * 26;
    }
}

File: Python/_171_Excel_Sheet_Column_Number/solution.py
class Solution:
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        if len(s) == 1:
            return ord(s[-1]) - 65 + 1
        return ord(s[-1]) - 65 + 1 + self.titleToNumber(s[:-1]) * 26
        